themes/classic/_dev/js/theme.js

```diff
@@ -30,12 +31,14 @@ for (var i in EventEmitter.prototype) {
 
 $(document).ready(() => {
   let dropDownEl = $('.js-dropdown');
+  const form = new Form();
```

Comment:
Because I only wrote this one. I was thinking const might be better in my case. I dont know for the other but I'm sure many of these vars would be fine in a `const`.